spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/IcebergPropertiesConverter.java

```diff
@@ -123,4 +126,19 @@ private void initJdbcProperties(
     icebergProperties.put(IcebergPropertiesConstants.ICEBERG_CATALOG_JDBC_USER, jdbcUser);
     icebergProperties.put(IcebergPropertiesConstants.ICEBERG_CATALOG_JDBC_PASSWORD, jdbcPassword);
   }
+
```

Comment:
could you add test in `TestIcebergPropertiesConverter`?

Reply:
fixed.